.github/workflows/handle-external-events.yml

```diff
@@ -169,3 +169,18 @@ jobs:
         run: "bash .ci-scripts/build-and-push-stdlib-documentation.bash $STDLIB_TOKEN"
         env:
           STDLIB_TOKEN: ${{ secrets.STDLIB_TOKEN }}
+
+  trigger-release-announcement:
```

Comment:
This is different than pretty much all our other apps.

There is nothing for the release that is built as an artifact on GitHub so, we wait here until we have received notification that the musl build of ponyc has been uploaded to Cloudsmith. That in turn results in a new standard library builder being built, then the standard library docs are updated and pushed and finally, once that is done, we trigger a release announcement.

Not all build artifacts might be completed at that point, but it is "good enough".